app/elements/routing.html

```diff
@@ -95,6 +95,12 @@
 					page.createTeam();
 					return;
 				}
+				if(qs[i].indexOf('invitedby=') === 0 && qs[i + 1].indexOf('teamname=') === 0) {
+					vInvitedBy = qs[i].split('=')[1];
+					vTeamName = qs[i + 1].split('=')[1];
```

Comment:
Same here with the global. Also, please avoid using [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation).